contracts/external/proxies/LiquidatorProxyV1ForSoloMargin.sol

```diff
@@ -108,16 +115,17 @@ contract LiquidatorProxyV1ForSoloMargin is
         public
         nonReentrant
     {
-        // validate the msg.sender and that the liquidAccount can be liquidated
-        checkRequirements(fromAccount, liquidAccount);
-
         // put all values that will not change into a single struct
         Constants memory constants = Constants({
             fromAccount: fromAccount,
             liquidAccount: liquidAccount,
-            minLiquidatorRatio: minLiquidatorRatio
+            minLiquidatorRatio: minLiquidatorRatio,
+            markets: getMarketsInfo()
```

Comment:
it's a little weird that these are in `constants` and not `cache` but I guess it's fine

Reply:
Yeah I guess constants is for stuff that doesn't change depending on the liquidation pair whereas cache is information that initializes for each liquidation market pair